src/wallet/wallet.h

```diff
@@ -1272,6 +1274,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     std::map<uint256, CWalletTx> mapWallet;
 
+    std::map<uint256, std::vector<RecipientMapping>> sendRecipients;
```

Comment:
This should go in the separate commit with the other unexplained changes (like the addition of `LoadRecipientMapping`).

Reply:
This is necessary to be able to look up recipient UAs by receiver. This had been left incomplete in #5549, and I hadn't realized it.